packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: fishtown-analytics/dbt_utils
-    version: [">=0.1.24", "<0.5.0"]
+    version: [">=0.4.0", "<0.6.0"]
```

Comment:
I think it was a mistake to leave the old lower bound here.
This technically makes it a `minor` release, but I also think we can get away with a patch release, since we're fixing a bug around version compatibility 👀

Reply:
I'm not convinced this was a mistake. Although adding `config-version: 2` to this package requires running dbt v0.17.0, both this package and dbt v0.17.0 could still handle a version-1-configured `dbt-utils`.

That said: I agree it's clearer this way, it makes our lives easier, and I can't imagine the person who would _actually_ be affected by this change. (They wouldn't be using dbt v0.17.0, so they couldn't be using >=0.1.0 of this package anyway!)

Reply:
Ah right, I understand. Do you think for this reason it should be a minor release instead of a patch release?